Fix download shell command

```sh
Warning: /opt/homebrew/bin is not in your PATH.
  Instructions on how to configure your shell for Homebrew
  can be found in the 'Next steps' section below.

==> Next steps:
- Run these three commands in your terminal to add Homebrew to your PATH:
    echo '# Set PATH, MANPATH, etc., for Homebrew.' >> /Users/akkeylab/.zprofile
    echo 'eval "$(/opt/homebrew/bin/brew shellenv)"' >> /Users/akkeylab/.zprofile
    eval "$(/opt/homebrew/bin/brew shellenv)"
```

```sh
Downloading ruby-3.2.0.tar.gz...
-> https://cache.ruby-lang.org/pub/ruby/3.2/ruby-3.2.0.tar.gz
Installing ruby-3.2.0...
ruby-build: using readline from homebrew
ruby-build: using gmp from homebrew

BUILD FAILED (macOS 13.2 using ruby-build 20230124)

Inspect or clean up the working tree at /var/folders/x4/krdh4qhs2gd4tsngh98n9dzm0000gn/T/ruby-build.20230129173537.19929.bUFEw9
Results logged to /var/folders/x4/krdh4qhs2gd4tsngh98n9dzm0000gn/T/ruby-build.20230129173537.19929.log

Last 10 log lines:
	Check ext/psych/mkmf.log for more details.
*** Fix the problems, then remove these directories and try again if you want.
Generating RDoc documentation
/private/var/folders/x4/krdh4qhs2gd4tsngh98n9dzm0000gn/T/ruby-build.20230129173537.19929.bUFEw9/ruby-3.2.0/lib/yaml.rb:3: warning: It seems your ruby installation is missing psych (for YAML output).
To eliminate this warning, please install libyaml and reinstall your ruby.
uh-oh! RDoc had a problem:
cannot load such file -- psych

run with --debug for full backtrace
make: *** [rdoc] Error 1
version 3.2.0 is not installed for ruby
ruby 2.6.10p210 (2022-04-12 revision 67958) [universal.arm64e-darwin22]
/Users/akkeylab/mac-auto-setup/setup.sh: line 138: where: command not found
unknown command: ruby. Perhaps you have to reshim?
```

```sh
/Users/akkeylab/mac-auto-setup/setup.sh: line 153: go: command not found
/Users/akkeylab/mac-auto-setup/setup.sh: line 154: where: command not found
/Users/akkeylab/.asdf/installs/golang/1.19.5/go/bin/go
/Users/akkeylab/mac-auto-setup/setup.sh: line 157: go: command not found
/Users/akkeylab/mac-auto-setup/setup.sh: line 158: go: command not found
/Users/akkeylab/mac-auto-setup/setup.sh: line 159: go: command not found
```

--- a/README.md
+++ b/README.md
@@ -2,7 +2,7 @@
 Auto setup system for macOS.
 
 ## macOS version
-macOS Monterey 12.2.1<br>
+macOS Ventura 13.2<br>
 Intel / Apple Silicon
 
 ## Downloads

--- a/app.sh
+++ b/app.sh
@@ -28,7 +28,7 @@ brew install --cask imageoptim
 brew install --cask visual-studio-code
 brew install --cask adobe-creative-cloud
 brew install --cask gyazo
-brew install --cask zoomus
+brew install --cask zoom
 brew install --cask discord
 brew install --cask notion
 brew install --cask figma

--- a/app.sh
+++ b/app.sh
@@ -12,6 +12,8 @@ EOS
 # Install web apps.
 #
 echo " ----- Install web apps ------"
+sudo softwareupdate --install-rosetta
+
 brew install --cask 1password
 brew tap AdoptOpenJDK/openjdk
 brew install --cask adoptopenjdk

--- a/download.sh
+++ b/download.sh
@@ -28,9 +28,10 @@ fi
 #
 # Install git
 #
-if ! command_exists git; then
+if [ ! -e "$(brew --prefix)/bin/git" ]; then
   echo " ------------ Git ------------"
   brew install git
+  brew info git
   git --version
   echo " ------------ END ------------"
 fi

--- a/download.sh
+++ b/download.sh
@@ -18,6 +18,9 @@ function command_exists {
 if ! command_exists brew; then
   echo " --------- Homebrew ----------"
   /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+  echo '# Set PATH, MANPATH, etc., for Homebrew.' >> ~/.zprofile
+  echo 'eval "$(/opt/homebrew/bin/brew shellenv)"' >> ~/.zprofile
+  source ~/.zprofile
   brew -v
   echo " ------------ END ------------"
 fi

--- a/settings/zsh/private.zsh
+++ b/settings/zsh/private.zsh
@@ -3,3 +3,12 @@ export LANG=en_US.UTF-8
 export SDKROOT="$(xcrun --sdk macosx --show-sdk-path)"
 
 setopt print_eight_bit
+
+# If you get a warning, do the following
+# compaudit | xargs sudo chmod g-w
+if type brew &>/dev/null; then
+  FPATH=$(brew --prefix)/share/zsh-completions:$FPATH
+
+  autoload -Uz compinit
+  compinit
+fi

--- a/setup.sh
+++ b/setup.sh
@@ -112,6 +112,7 @@ cp $(
   cd $(dirname ${BASH_SOURCE:-$0})
   pwd
 )/settings/zsh/p10k.zsh ~/.yadr/zsh/p10k.zsh
+source ~/.zshrc
 echo " ------------ END ------------"
 
 #
@@ -121,6 +122,7 @@ if ! command_exists asdf; then
   echo " ----------- asdf ------------"
   brew install asdf
   echo -e "\n. $(brew --prefix asdf)/libexec/asdf.sh" >>~/.yadr/zsh/private.zsh
+  source ~/.zshrc
   echo " ------------ END ------------"
 fi
 
@@ -219,6 +221,7 @@ case $input in
   echo "export GITHUB_ACCESS_TOKEN=${token}" >>~/.yadr/zsh/private.zsh
   echo "export HOMEBREW_GITHUB_API_TOKEN=${token}" >>~/.yadr/zsh/private.zsh
   echo "Writing to ~/.yadr/zsh/private.zsh is complete."
+  source ~/.zshrc
   echo " ------------ END ------------"
   ;;
 esac

--- a/setup.sh
+++ b/setup.sh
@@ -271,3 +271,14 @@ case $input in
   )/app.sh
   ;;
 esac
+
+read -p 'Would you like to create the recommended development directory ? [y/n]' input
+case $input in
+'' | [Nn]*)
+  echo "Skip"
+  ;;
+[Yy]*)
+  mkdir Development
+  echo " ------------ END ------------"
+  ;;
+esac

--- a/setup.sh
+++ b/setup.sh
@@ -131,6 +131,12 @@ fi
 #
 if [ ! -e "$(echo ~$USERNAME)/.asdf/shims/ruby" ]; then
   echo " ----------- Ruby ------------"
+  # No longer bundle 3rd party sources
+  # https://www.ruby-lang.org/en/news/2022/12/25/ruby-3-2-0-released
+  brew install libyaml ruby-build
+  echo -e "export RUBY_CONFIGURE_OPTS=\"--with-openssl-dir=$(brew --prefix openssl@1.1)\"" >>~/.yadr/zsh/private.zsh
+  source ~/.zshrc
+
   asdf plugin add ruby https://github.com/asdf-vm/asdf-ruby.git
   ruby_latest=$(asdf list all ruby | grep -v '[a-z]' | tail -1 | sed 's/ //g')
   asdf install ruby $ruby_latest

